### PR TITLE
Add several missing upgrade grants

### PR DESCRIPTION
**Pull request by spiegela**
Pretty self-explanatory, I think. Definitely let me know if there are any questions or suggestions.

### CODE REVIEW

File: data/upgrades.js [hunk 1]
```diff
@@ -2545,7 +2585,17 @@
...
+    "grants": [
+      {
+        "type": "slot",
+        "name": "Crew"
+      },
+      {
+        "type": "slot",
+        "name": "Crew"
+      }
+    ]
```
Comment (guidokessels) [inline data/upgrades.js]: Mhm perhaps we should also have a way to define the slots that get removed. The TIE Shuttle card should remove all Torpedo, Missile and Bomb slots.

File: data/upgrades.js [hunk 2]
```diff
@@ -1800,6 +1810,18 @@
...
+      },
+      {
+        "type": "slot",
+        "name": "Torpedo"
+      },
+      {
+        "type": "slot",
+        "name": "Missile"
+      },
+      {
```
Comment (guidokessels) [inline data/upgrades.js]: This is technically not correct. You will get _one_ of the slots, not all of them. I think we should have a way of defining a choice of slots. But until then, could you remove the `grants` field for this card? I'm happy to merge the rest of the changes :)

File: data/upgrades.js (hunk 1 above)
Comment (spiegela) [inline data/upgrades.js]: Yeah, I thought about adding a "revokes" property on the upgrade as well, but I figured I should make one change at a time.

If you like that approach, though, I can do that with this PR.

File: data/upgrades.js (hunk 2 above)
Comment (spiegela) [inline data/upgrades.js]: Yeah, I haven't seen a lot of upgrade cards with choices like that. Maybe, we could add an `or` or `oneOf` property to the grant.  Might look something like:

```
    "grants": [
      {
        "oneOf": [
          {
            "type": "slot",
            "name": "Cannon",
          },
          {
            "type": "slot",
            "name": "Missle",
          },
          {
            "type": "slot",
            "name": "Torpedo",
          }
        ]
      }
    ]
```

Comment (guidokessels) [inline data/upgrades.js]: Lets worry about that in a separate pr. Could you remove the `grants` from `Heavy Scyk` so I can merge this? :)

**Comment by guidokessels** [comment]
Thanks @spiegela! Apologies for the delay, I was away on holidays :)

**Comment by spiegela** [comment]
Sweet.  Thanks.

**Pull request merged**